src/Encoding.ts

```diff
@@ -1,31 +1,31 @@
-'use strict';
```

Comment:
‘use strict’ is a thing for jshint?  So that reminds me that we need a linter for ts instead.